.github/workflows/python-publish.yml

```diff
@@ -1,16 +1,28 @@
-name: Upload Python Package
+name: Publish Release
 
 on:
   release:
-    types:
-      - created
+    types: [created]
```

Comment:
I think this should be `released`.

Reply:
I think this should be working for pre-release also. So created is a lot better.

Reply:
Or `published`?